samples/samples/admin/samples.py

```diff
@@ -61,16 +61,94 @@ def create_instance(instance_id):
 # [END spanner_create_instance]
 
 
+# [START spanner_create_instance_with_processing_units]
+def create_instance_with_processing_units(instance_id, processing_units):
+    """Creates an instance."""
+    spanner_client = spanner.Client()
+
+    config_name = "{}/instanceConfigs/regional-us-central1".format(
+        spanner_client.project_name
+    )
+
+    request = spanner_instance_admin.CreateInstanceRequest(
+        parent="projects/{}".format(spanner_client.project),
+        instance_id=instance_id,
+        instance=spanner_instance_admin.Instance(
+            config=config_name,
+            display_name="This is a display name.",
+            processing_units=processing_units,
+            labels={
+                "cloud_spanner_samples": "true",
+                "sample_name": "snippets-create_instance_with_processing_units",
+                "created": str(int(time.time())),
+            },
+        ),
+    )
+
+    operation = spanner_client.instance_admin_api.create_instance(request=request)
+
+    print("Waiting for operation to complete...")
+    instance = operation.result(OPERATION_TIMEOUT_SECONDS)
+
+    print(
+        "Created instance {} with {} processing units".format(
+            instance_id, instance.processing_units
+        )
+    )
+
+
+# [END spanner_create_instance_with_processing_units]
+
+
+# [START spanner_create_database]
+def create_database(instance_id, database_id):
+    """Creates a database and tables for sample data."""
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+
+    request = spanner_database_admin.CreateDatabaseRequest(
+        parent=instance.name,
```

Comment:
I I don't see `name` as a property of handwritten [instance](https://github.com/googleapis/python-spanner/blob/3aab0ed5ed3cd078835812dae183a333fe1d3a20/google/cloud/spanner_v1/instance.py#L114) class. I don't understand from where this variable is getting assigned.

However, we should directly use `instance_id ` instead of handwritten ones.